sale_financial_risk/models/sale.py

```diff
@@ -69,42 +66,25 @@ def _compute_amount_to_invoice(self):
             where
                 `amount_invoiced` is taxed amount previously explained
         """
-        refund_lines_product = self.env['account.invoice.line']
-        for line in self:
+        for line in self.filtered(lambda l: l.state == 'sale'):
```

Comment:
¿Por qué actualizar solo las SOLines  que están en estado 'sale'? Yo pienso que deberíamos mantenerlo como estaba: 
for line in self:
¿Qué crees?

Reply:
Las lineas a las que no se les establece un valor específico en un compute se ponen a 0, por lo que realmente se computan todas, pero las que no tienen estado sale no cuentan para el riesgo financiero.
Descartamos las líneas en estado 'done' porque es un mecanismo para que no se tengan en cuenta para el riesgo aunque el pedido no se haya servido completamente.

Reply:
Se computan todas las que vienen en el self